hub/api/dataset.py

```diff
@@ -207,3 +207,9 @@ def from_path(path: str):
             "Automatic dataset ingestion is not yet supported."
         )  # TODO: hub.auto
         return None
+
+    def __str__(self):
```

Comment:
should probably also have the len here

Reply:
@AbhinavTuli What do you mean? `len(ds)` is still up for debate and the slicing information is given by `index_str`

Example:
```
>>> print(str(ds))
Dataset(mode='a', index=Index([slice(1, 2, 1)]), tensors=['image'])